libutils/buffer.c

```diff
@@ -427,38 +427,6 @@ int BufferPrintf(Buffer *buffer, const char *format, ...)
     return printed;
 }
 
-int BufferVPrintf(Buffer *buffer, const char *format, va_list ap)
```

Comment:
I think we should leave this one. It's part of the API of Buffer, and this might be used later.